spec/rails_helper.rb

```diff
@@ -54,18 +54,30 @@
   end
 end
 
-Capybara.register_driver :chrome do |app|
-  Capybara::Selenium::Driver.new(app, browser: :chrome)
-end
+# Capybara.register_driver :chrome do |app|
+#   Capybara::Selenium::Driver.new(app, browser: :chrome)
+# end
+
+# Capybara.register_driver :headless_chrome do |app|
+#   capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+#     chromeOptions: { args: %w[headless disable-gpu] }
+#   )
 
-Capybara.register_driver :headless_chrome do |app|
+#   Capybara::Selenium::Driver.new app,
+#                                  browser: :chrome,
+#                                  desired_capabilities: capabilities
+# end
```

Comment:
can we remove it, if this is not needed anymore?

Reply:
removed